packages/node-cli/src/commands/object/update/UpdateOutputHandler.js

```diff
@@ -20,8 +21,18 @@ module.exports = class UpdateOutputHandler extends BaseOutputHandler {
 	}
 
 	parse(actionResult) {
-		const updatedObjects = actionResult.data.filter((element) => element.type === UPDATED_OBJECT_TYPE.SUCCESS);
-		const noUpdatedObjects = actionResult.data.filter((element) => element.type !== UPDATED_OBJECT_TYPE.SUCCESS);
+		this._parseUpdateObjects(actionResult);
+		this._parseUpdateCustomRecordsWithInstancesNonInteractiveExecution(actionResult);
+		return actionResult;
```

Comment:
I think this is the only command that outputs messages during the action itself which is a "kind of hack" for the CustomRecordWithInstances and it will be not obvious to someone looking at this code.
My suggestion is to at least add a comment here before line 24, something like: // updated CustomRecordsWithInstances had been logged during the action execution.

Then I also think that we could rename `_parseUpdateCustomRecordsWithInstancesNonInteractiveExecution` to `_showNonInteractiveExecForCustomRecordWithInstances` or `_logNonInteractiveExecForCustomRecordWithInstances` because we are not parsing/outputting any part of the actionResult.

The rest of the file looks good to me.